src/utils.tsx

```diff
@@ -152,3 +153,28 @@ export function isOptionFacet(
 ): facet is PlpMultipleFacet | PlpSingleFacet | PlpHierarchicalFacet {
   return isMultipleOrBucketedFacet(facet) || isSingleFacet(facet) || isHierarchicalFacet(facet);
 }
+
+export function getSearchCnstrcDataAttributes(
+  data: Nullable<PlpSearchData>,
+  initialResponse: SearchResponse | undefined,
```

Comment:
wdyt of just taking in `data` here? `initialResponse` would already be passed into the `useSearchResults` hook, so it would reflect in `data` anyways if it was provided.

Reply:
Good callout I missed that